docker/ubuntu-indexer/bin/system-setup.sh

```diff
@@ -23,7 +23,9 @@ apt_packages+=('libclang-dev')
 apt_packages+=('llvm')
 apt_packages+=('llvm-dev')
 apt_packages+=('npm')
-apt_packages+=('nodejs')
+
+# Dependency for nodejs
+apt_packages+=('apt-transport-https')
```

Comment:
curl is already listed on [line 16](https://github.com/pelmers/dxr-docker/blob/a9d354d802e21141f9b2a7bd7156e52182d4264a/docker/ubuntu-indexer/bin/system-setup.sh#L16) of the file. (did you want to make sure we install curl?)